src/pages/home/components/nav-bar.module.scss

```diff
@@ -0,0 +1,61 @@
+.navBar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: sticky;
+  top: 0;
+  width: 100%;
+  gap: 10%;
+  background-color: white;
+  height: 60px;
+  border-bottom: solid;
+  border-width: 1px;
+  border-color: #cecece;
+  font-size: large;
+
+  & form {
+    width: 300px;
+  }
+
+  & input {
+    width: 100%;
+    padding-left: 40px;
+    border-radius: 10px;
+    background-color: #ededed;
+    border: none;
+    height: 40px;
+  }
+
+  & input:focus{
+    outline: none !important;
+    border:1px solid #9c9c9c;
+  }
+}
+
+.webName {
+  width: 300px;
+  background: #121FCF;
+  background: linear-gradient(to right, #121FCF 0%, #CF1512 100%);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: x-large;
+  font-weight: bold;
+  text-align: center;
+}
+
+.wrapper {
+  position:relative;
+  width: 100%;
+  color: #9c9c9c;
```

Comment:
Wouldn't it be better define the colors in colors.scss and use it as a variable here ??